entrypoint.sh

```diff
@@ -25,9 +25,13 @@ if [ -f /extra-entrypoint-run-first.sh ]; then
     source /extra-entrypoint-run-first.sh
 fi
 
-# Copy LocalSettings from share if exists
+# check if shared LocalSettings exists already
 if [ -e "/shared/LocalSettings.php" ]; then
-  cp /shared/LocalSettings.php /var/www/html/LocalSettings.php
+    # do not install wiki, just update LocalSettings from template
+    export DOLLAR='$'
+    envsubst < /LocalSettings.php.template > /var/www/html/LocalSettings.php
```

Comment:
I don't fully understand this, and it is not clear which problem you are trying to solve, however, I doubt that this will work as it will always be recreated shared config and thus dbpasswords and secrets won't be picked up correctly, since they can not be regenerated.

Reply:
The problem is that when changes are introduced in LocalSettings.php.mardi.template, like in https://github.com/MaRDI4NFDI/docker-wikibase/pull/13/files, these will never be deployed, since LocalSettings.php is always copied from /shared/. The file has to be modified manually on the server.

Reply:
but then this answers my question here. https://github.com/MaRDI4NFDI/docker-wikibase/issues/21#issuecomment-1123347604
apparently the way to go is to just document this clearly and stick with the current version

Reply:
Yes, I think updating it on the server is the way to go. LocalSettings should not be modified automatically after it was created initially. At least that is the way how I understand the philosophy of that file.